flow/flow.yaml

```diff
@@ -29,27 +29,47 @@ components:
           minItems: 1
           items:
             $ref: './packet-headers/header.yaml#/components/schemas/Flow.Header'
+          x-status:
+            status: deprecated
+            information: |- 
+              Use flow.ingress_packet instead.
           x-field-uid: 2
+        ingress_packet:
```

Comment:
From previous review:  https://github.com/open-traffic-generator/models/pull/281
We have a scenario where for Ingress tracking, it is sometimes useful if we have a generic Flow Descriptor field which contains autogenerated unique string describing endpoint and then user can track Tx/Rx/Drop per such sub-flow in one flow/stream. This is useful for example when 10000 RSVP tunnels are aggregated into single stream where tracking by MPLS label is not very useful ( assigned by DUT and difficult to track MPLS label -> ingress tunnel + MPLS label can change at run-time e.g. Dynamic MPLS ). So a auto tracking option where sub-flows are automatically assigned names and optional per sub-flow tracking is provided can be useful. We could create one traffic item per tunnel but that gets limited by max flows/stream per port ( say mostly upto 256 for h/w) . This could also be applicable where tracking is done by autogenerated names .